Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+
+FROM golang:1.21
```

Comment:
Please use a multi-stage Dockerfile:

1. Stage one, from “golang:1.21-alpine” just builds the container
2. Stage 2 is the production stage and it’s based on distroless.

Example: https://github.com/GoogleContainerTools/distroless/blob/main/examples/go/Dockerfile

Reply:
thanks for this! Updated!